lib/prompts.js

```diff
@@ -1,20 +1,44 @@
-module.exports.modulesDir = {
-  type: 'input',
-  name: 'modulesDir',
-  message: 'Where is your "modules" directory, relative to project root?',
-  default: 'src/modules',
-  store: true,
-};
+const R = require('ramda');
 
-module.exports.moduleName = {
-  type: 'input',
-  name: 'moduleName',
-  message: 'Module name?',
-  store: true,
+module.exports.prompts = {
+  modulesDir: {
+    type: 'input',
+    name: 'modulesDir',
+    message: 'Where is your "modules" directory, relative to project root?',
+    default: 'src/modules',
+    store: true,
+  },
+  moduleName: {
+    type: 'input',
+    name: 'moduleName',
+    message: 'Module name?',
+    store: true,
+  },
+  componentName: {
+    type: 'input',
+    name: 'componentName',
+    message: 'Component name?',
+  },
 };
 
-module.exports.componentName = {
-  type: 'input',
-  name: 'componentName',
-  message: 'Component name?',
+module.exports.promptsOrder = ['modulesDir', 'moduleName', 'componentName'];
+
+const translateRequiredProp = (prop) => {
+  switch (prop) {
+    case 'moduleNameUpperFirst': {
```

Comment:
Wish the linter would yell at me. The switch used to do more and the brackets are relics, thanks for the note.